### ISSUE TITLE
Reactant not exists.

### ISSUE THREAD

**Issue by Marchiuzzz** [cybercog/laravel-love#182]
Hi, I have followed the setup guidelines to get this package installed for my existing project with already existing models and records. However, I ran into a problem I cannot seem to solve. Whenever I try to use "$reacterFacade->reactTo" I get this "Reactant not exists." error, I have registered reacterers as well as reactants manually, as suggested in https://github.com/cybercog/laravel-love/issues/105 , but that did not solve the issue. Here's my simple function
```
public function like(Request $request){
      if(Auth::check()){
        $user = Auth::user();
        $object = DynamicAbstractObject::where('unique_entity_id', $request->input('entity'))->first();
        if(!empty($object)){
          if($object->isNotRegisteredAsLoveReactant()){
            $object->registerAsLoveReactant();
            $object->save();
          }
          if($user->isNotRegisteredAsLoveReacter()){
            $user->registerAsLoveReacter();
            $user->save();
          }
          $reacterFacade = $user->viaLoveReacter();
          $reacterFacade->reactTo($object, 'Like', 4.0);
        }
      }
    }
```
![image](https://user-images.githubusercontent.com/73941456/99534755-970b1180-29b0-11eb-9443-2880e4c800bf.png)

Any help would be much appreciated

**Comment by antonkomarev** [comment]
Do you have records in `love_reactants` database table? There is no need to create reactants on the fly on each `like` call. It should be done once for each model on initial package integration: https://laravel-love.readme.io/docs/restore-missing-reactants

**Comment by Marchiuzzz** [comment]
I found what the problem was. I ignored the fact that when executing this step of setup "php artisan love:setup-reactable --model="App\Comment" I ran the migration and the column was created but foreign key failed to create. All I had to do was run "php artisan love:setup-reactable --model="App\Comment --nullable" instead and then repeat the manual registration. Everything works now, thanks for this wonderful package.
